subhub/hub/stripe/customer.py

```diff
@@ -98,13 +65,44 @@ def run(self):
 class StripeCustomerSubscriptionCreated(AbstractStripeHubEvent):
     def run(self):
         logger.info("customer subscription created", payload=self.payload)
+        invoice_number = None
+        invoice_id = None
+        charge = None
+        last4 = None
+        brand = None
```

Comment:
is there a point to setting these values to None?  If not, delete these assignments.